spec/search_people_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+RSpec.describe "Test Search when user exist" do
+
+
+	let(:email) {"adneela@hotmail.com"}
+	let(:password) {"adnela"}
+	let (:ime) {"Adnela Cavcic"}
+	let(:trazi) {"fakultetski mail"}
```

Comment:
Use variables that make sense. Always consider that someone might have to maintain your tests, and you want to make them as readable as possible.